src/colorizer/ColorizeCanvas.ts

```diff
@@ -80,7 +80,7 @@ const getDefaultUniforms = (): ColorizeUniforms => {
   const emptyFeature = packDataTexture([0], FeatureDataType.F32);
   const emptyOutliers = packDataTexture([0], FeatureDataType.U8);
   const emptyInRangeIds = packDataTexture([0], FeatureDataType.U8);
-  const emptyColorRamp = new ColorRamp(["black"]).texture;
+  const emptyColorRamp = new ColorRamp(["#aaa", "#fff"]).texture;
```

Comment:
This should never be visualized but added a nice neutral grey texture instead of a harsh black.